adapters/algorix/algorixtest/supplemental/check_first_imp.json

```diff
@@ -0,0 +1,123 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "banner": {
+          "format": [{"w": 320, "h": 50}]
+        },
+        "ext": {
+          "bidder": {
+            "sid": "testSid1",
+            "token": "testToken1"
+          }
+        }
+      },
+      {
+        "id": "test-imp-id_2",
+        "banner": {
+          "format": [{"w": 300, "h": 250}]
+        },
+        "ext": {
+          "bidder": "aaa"
+        }
+      }
```

Comment:
Prebid Server code would filter out the second imp in this test case but it doesn't hurt to include this test case.

Reply:
we delete this test check file.